config/environments/development.rb

```diff
@@ -32,6 +32,7 @@
 
     config.cache_store = :null_store
   end
+  config.session_store :cookie_store, key: LaaEstimateFinancialEligibilityForLegalAid::SESSION_COOKIE_NAME
```

Comment:
So I _keep_ hitting the CookieOverflow error in development, and I would personally prefer the dev environment to use Redis to avoid having to clear cookies and restart the flow every couple of days.

(Also, if we used Redis in dev as well, we could explore things like storing the CFE result in the session, obviating the need to call CFE _twice_ just to print the results page. At the moment the result payload is big enough to explode the cookie).

If it feels like too much faff to have working Redis in dev I'm OK to be overruled - but maybe for future devs we could add a line in the readme explaining that CookieOverflows are a dev-environment-specific error, and offering a workaround?

Reply:
I don't really use the dev environment personally, it seems a bit overkill to have a different server running just for this purpose. I've added a commit to use the memory_store for development which seems to work well for me (and I think I've used this before in MoJ), could you try it out @patrick-laa and see if it works for you?

Reply:
@starswan great shout. Yes, that works for me, and I'm confident that the differences in environment between dev/test/prod are small enough that it won't come back to haunt us, and that by saying it explicitly out loud like this I'm not immediately jinxing it.

Reply:
@patrick-laa There are a few minor differences (from experience) but it's fine as long as you stick to primitive types and don't try to serialise/deserialise whole objects